### DISCUSSION TITLE
be able to identify when a sensor is not updating

### DISCUSSION THREAD

**Discussion by quilmore** [frenck/spook#487] [Ideas]
I have some esphome sensors connecting to the data source by BT, at times for whatever reason the BT link is lost and doesn't recover for several minutes/hours
I'd like something that spots no change (when there should have been change) and act on it (send notification, trigger an action, etc) HA does not have something natively to do do that, thanks

**Comment by frenck** [comment]
It does... you can check the last changes timestamp on those sensors.
